smoke-tests/images/grpc/build.gradle.kts

```diff
@@ -0,0 +1,60 @@
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+
+plugins {
+  java
+
+  id("com.google.cloud.tools.jib") version "3.2.1"
+  id("com.diffplug.spotless") version "6.4.2"
+}
+
+group = "io.opentelemetry"
+version = "0.0.1-SNAPSHOT"
+
+repositories {
+  mavenCentral()
+}
+
+spotless {
+  java {
+    googleJavaFormat()
+    licenseHeaderFile(rootProject.file("../../../buildscripts/spotless.license.java"), "(package|import|public)")
+  }
+  kotlinGradle {
+    ktlint().userData(mapOf("indent_size" to "2", "continuation_indent_size" to "2", "disabled_rules" to "no-wildcard-imports"))
+    target("**/*.gradle.kts")
+  }
+}
+
+dependencies {
+  implementation(platform("io.grpc:grpc-bom:1.45.1"))
+  implementation(platform("io.opentelemetry:opentelemetry-bom:1.0.0"))
```

Comment:
I left this at 1.0.0 since it seems to serve a purpose of verifying the javaagent with the old API version